### ISSUE TITLE
Guidelines for requesting repos added to the org

### ISSUE THREAD

**Issue by steelbrain** [AtomLinter/Meta#3]
Just like the title says

**Comment by DanPurdy** [comment]
Yeah I was wondering about this, do you mean actually adding to the Github organisation, to the linter website or both?

Something simple such as a checklist consisting of
- a licence
- contribution guidelines
- eslint / coffee-lint config file
- a README with acceptable instructions
- possibly some community guidelines such as with the Sass community [guidelines](http://sass-lang.com/community-guidelines)

**Comment by steelbrain** [comment]
You don't necessarily have to be in the organization to be added to the website. I meant the AtomLinter org.

**Comment by DanPurdy** [comment]
In which case instructions to cover both would be good. I'm assuming a PR to get a plugin added to the site would be enough but the above list would be a good default guideline for having your plugin added to the organisation.

A quick breakdown of the steps needed to be taken to transfer ownership would also be ideal. Whether that's liaising to have a new blank repo setup that you have push access to or whatever.

**Comment by SpainTrain** [comment]
> possibly some community guidelines such as with the Sass community guidelines

:+1: 

http://todogroup.org/opencodeofconduct/ is a nice easy one, and is already used by Atom

**Comment by SpainTrain** [comment]
What are the concrete steps to make this so?  Add the info to the AtomLInter.org site?

**Comment by SpainTrain** [comment]
Could also see something like http://eslint.org/docs/developer-guide/governance being useful for clarity and transparency. Our model is obviously more distributed, so would take some work to draft unfortunately

**Comment by Arcanemagus** [comment]
I could see our model fitting in there, each repository having a Project Lead (ower in the team), only a few have enough people to have more than that role though.

**Comment by SpainTrain** [comment]
That's almost certainly the most useful of any role we would want to canonize.  I often have no clue whether a linter has an active primary maintainer (until sifting through closed and/or recent issues/PRs)

**Comment by Arcanemagus** [comment]
Yea, it would be nice to make more usage of the teams within the org.
